sql/sqlite/tables.sql

```diff
@@ -1,32 +1,59 @@
+
+-- labki_content_repo
+--  └── labki_content_ref
+--          └── labki_pack
+--                  └── labki_page
+
 -- ===========================================================
---  labki_content_repo: top-level content repository
+--  labki_content_repo: top-level content repository (bare clone)
 -- ===========================================================
-CREATE TABLE IF NOT EXISTS labki_content_repo (
-  content_repo_id INTEGER PRIMARY KEY AUTOINCREMENT,
-  content_repo_url TEXT NOT NULL,
-  content_repo_name TEXT,
-  default_ref TEXT,
-  created_at INTEGER,
-  updated_at INTEGER,
-  UNIQUE (content_repo_url)
+CREATE TABLE labki_content_repo (
+  content_repo_id     INTEGER PRIMARY KEY AUTOINCREMENT,
+  content_repo_url    TEXT NOT NULL UNIQUE,   -- canonical repository URL (e.g., https://github.com/Aharoni-Lab/labki-packs)
+  content_repo_name   TEXT,                   -- manifest-defined or inferred name
+  default_ref         TEXT DEFAULT 'main',    -- repo’s default branch
+  bare_path           TEXT,                   -- filesystem path to the bare clone
+  last_fetched        INTEGER,                -- timestamp of last git fetch
+  created_at          INTEGER NOT NULL,
+  updated_at          INTEGER NOT NULL
+);
+
+-- ===========================================================
+--  labki_content_ref: per-branch/tag/commit (worktree level)
+-- ===========================================================
```

Comment:
i think i'm a little confused on the structure, so `labki_content_repo` is a `1:many` relationship to `labki_content_ref`? or is it `1:1`? are we checking out the same repository multiple times into multiple directories? or what is the expected workflow. I sort of expected one clone, and then e.g. when it was time to make a PR, the extension makes a branch, commits changes, pushes branch, and then switches back to `main` (or whatever the configured ref is). why multiple clones?